.eslintignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
Not really needed unless pointed by eslint to lint. Can be removed

Reply:
Let's remove it.